src/xlsx/xlsxzipreader.cpp

```diff
@@ -48,7 +48,11 @@ ZipReader::~ZipReader()
 
 void ZipReader::init()
 {
+#if QT_VERSION >= 0x050600
+    QVector<QZipReader::FileInfo> allFiles = m_reader->fileInfoList();
```

Comment:
You don't need to test QT_VERSION. fileInfoList() always returns QVector.  It wouldn't be a bad idea to make this const.